build.gradle

```diff
@@ -313,7 +313,7 @@ subprojects {
 
     doFirst {
       def versionOverrides = [
-//        project(':internal:venice-common').file('src/main/resources/avro/StoreVersionState/v5', PathValidation.DIRECTORY)
+        project(':internal:venice-common').file('src/main/resources/avro/StoreVersionState/v27', PathValidation.DIRECTORY),
```

Comment:
Use this path instead: `internal/venice-common/src/main/resources/avro/StoreMetaValue/v28/StoreMetaValue.avsc`